cmd/cosign/cli/options/signblob.go

```diff
@@ -49,4 +50,7 @@ func (o *SignBlobOptions) AddFlags(cmd *cobra.Command) {
 
 	cmd.Flags().StringVar(&o.Output, "output", "",
 		"write the signature to FILE")
+
+	cmd.Flags().StringVar(&o.PubKeyOutput, "pubkey-output", "",
```

Comment:
What about changing this to `--output-cert`?

Reply:
@Dentrax I was a little bit confused about cert and public key... because this does not need to be a certificate it can also be a pubkey, IIRC.

@asraa are you fine with changing this to --output-cert instead of `-pubkey-output`?

Thoughts? @dlorenc

Reply:
I think it should always be a cert for the keyless modes right?

Reply:
Yeah, i think that's more clear.